src/parallel/DomainDecompMPIBase.h

```diff
@@ -106,7 +107,31 @@ class DomainDecompMPIBase: public DomainDecompBase {
 	}
 
 	void collCommAllreduceSumAllowPrevious() override {
+	    // TODO: Write Timer in here; or in DomainDecompBase
+#if defined(ENABLE_MPI)
+        Timer mpi_ReductionAll_timer;
+	    mpi_ReductionAll_timer.start();
+#endif
 		_collCommunication->allreduceSumAllowPrevious();
+#if defined(ENABLE_MPI)
+	    mpi_ReductionAll_timer.stop();
+	    double runtime = mpi_ReductionAll_timer.get_etime();
+
+	    MPI_Comm_size(_comm, &size );
+        MPI_Comm_rank(_comm, &rank );
+
+        _processTimer.insertTime(int(rank), double(runtime));
+        _processTimer.printTimeOnConsole();
+
+	    //MPI_File_open(_comm, "processRuntime.txt", MPI_MODE_WRONLY, MPI_INFO_NULL, &_logFH);
+	    //std::cout << "Rank: " << rank << " Size: " << size << " Runtime: " << runtime << " seconds" << std::endl;
+        //buf = (int *)malloc( size * sizeof(int) );
+        //buf[0] = rank;
+	    //MPI_File_write_ordered( _logFH, buf, 1, MPI_INT, &status );
+        //MPI_File_close(&_logFH);
+
+
+#endif
```

Comment:
please try to keep the formatting consistent in the lines you edit, for mardyn this means indentation by tabs ;)